caddy-builder.sh.tmpl

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -eu
+
+args=""
+for p; do
+	args="$args --with $p"
+done
+
+/usr/bin/xcaddy build --output /usr/bin/caddy $args
```

Comment:
As noted in the previous comment, should use `$CADDY_VERSION` here I think.

Reply:
`CADDY_VERSION` is already set, and `xcaddy` interprets it automatically - see https://github.com/caddyserver/xcaddy#environment-variables

> The `xcaddy` command will use the latest version of Caddy by default. You can customize this for all invocations by setting the `CADDY_VERSION` environment variable.

Reply:
Neat! Could still use a comment though, I guess.